test/policies/oauth/jwt-token.js

```diff
@@ -14,8 +15,8 @@ const serverHelper = require('../../common/server-helper');
 const config = require('../../../lib/config');
 const testHelper = require('../../common/routing.helper')();
 
-const originalGatewayConfig = JSON.parse(JSON.stringify(config.gatewayConfig));
-const originalSystemConfig = JSON.parse(JSON.stringify(config.systemConfig));
+const originalGatewayConfig = clone(config.gatewayConfig);
```

Comment:
just use new Config({}) to create your own config. copying some implicit config can result in hard track error if this default config will be changed

Reply:
Oh interesting. Let's see if I can make it work with your suggestion.